### PR TITLE
Docker sql

### LINKED ISSUE #4: 環境架設
讓所有人可以跑起專案

### PR DESCRIPTION
**Pull request by nonorze-VOXX**

### CODE REVIEW

**Review by huhuhu0420 — DISMISSED** [review]

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+services:
+  postgres_db:
+    image: postgres:latest
+    restart: always
+    ports: 
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=$DB_USER
+      - POSTGRES_PASSWORD=$DB_PASSWORD
```
Comment (NOOBDY) [inline docker-compose.yaml]:
```suggestion
    env_file:
      - .env
```

File: README.md [hunk 2]
```diff
@@ -1,30 +1,68 @@
...
+
+```bash
+cd sql
+docker-compose up --detach
+```
+
+### into postgres
+
+```bash
+psql -h localhost -p 5432 -U user
```
Comment (NOOBDY) [inline README.md:57]: use `docker exec -it <container> psql ...` so users don't have to install postgres on their machine

File: README.md [hunk 3]
```diff
@@ -1,30 +1,68 @@
...
 not yet use
+
 ```bash
+
 ```
+
 ### run
+
 ```bash
 go run .
```
Comment (NOOBDY) [inline README.md:38]: don't you need `go get`?

File: README.md (hunk 2 above)
Comment (nonorze-VOXX) [reply README.md:57]: listen good to can add new line to readme

File: README.md (hunk 3 above)
Comment (nonorze-VOXX) [reply README.md:38]: it look no need now
maybe huhuhu pr will add it

File: README.md (hunk 2 above)
Comment (nonorze-VOXX) [reply README.md:57]: fixed

File: docker-compose.yaml (hunk 1 above)
Comment (nonorze-VOXX) [reply docker-compose.yaml]: fixed

File: README.md (hunk 3 above)
Comment (nonorze-VOXX) [reply README.md:38]: fixed

**Review by huhuhu0420 — APPROVED** [review]

**Pull request merged**